docs/JK.Common/TypeHelpers/RegexHelper.md

```
@@ -0,0 +1,597 @@
+                ○ 7th capture group.
+                    ○ Match a character in the set [#-'+-9;?A-Z\\_a-z\u212A] atomically any number of times.
+            ○ Match if at the end of the string or if before an ending newline.
+```
+### ZipCodeRegex
+
+**Signature:** ``ZipCodeRegex()``
+
+**Summary:**
+
+**Remarks:** Pattern: 
+ ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)